posts/parent-class/index.qmd

```diff
@@ -0,0 +1,50 @@
+---
+title: "Choosing the Right Parent for R Object Classes"
+author:
+  - name: "Hugo Gruson"
+    orcid: "0000-0002-4094-1476"
+date: "2024-06-24"
+categories: [R, R package, object-oriented programming, S3, interoperability]
+---
+
+[I have recently published](https://hugogruson.fr/posts/compa-tibble/) a [series of blog posts](https://epiverse-trace.github.io/posts/s3-generic/) on the reasons why one may want to start using object-oriented programming (and more specifically R S3 classes) to improve interoperability with other tools from the ecosystem.
```

Comment:
Could you rather provide summary texts of the posts in square brackets?

For example, I have recently published about [making your inputs compatible with tibbles if they expect data.frames](URL) and [how to convert your functions to S3](URL) in a few steps.

Reply:
The goal here is to focus on the common topic across past posts, which helps justify this post.

But I think this is a good recommendation in general and will try to apply it wherever possible in the future.